analytics/views.py

```diff
@@ -1171,6 +1171,9 @@ def support(request: HttpRequest) -> HttpResponse:
             elif sponsorship_pending == "false":
                 update_sponsorship_status(realm, False)
                 context["message"] = f"{realm.string_id} is no longer pending sponsorship."
+            event_time = timezone_now()
+            RealmAuditLog.objects.create(realm=realm, acting_user=request.user, event_type=RealmAuditLog.REALM_SPONSORSHIP_STATUS_CHANGED,
```

Comment:
Move this to `update_sponsorship_status` and make it take `acting_user` as an optional argument.

The event name can be `REALM_SPONSORSHIP_PENDING_STATUS_CHANGED`

Also include the old and new status(true or false) in extra_data. It can be